specification/analysisservices/resource-manager/Microsoft.AnalysisServices/2017-07-14/analysisservices.json

```diff
@@ -660,7 +660,7 @@
           ],
           "x-ms-enum": {
             "name": "Status",
-            "modelAsString": true
+            "modelAsString": false
```

Comment:
@vishrutshah Shouldn't the breaking change tool flag this?  This changes the type returned in the model.